scapy/arch/windows/__init__.py

```diff
@@ -264,6 +264,16 @@ def test_windump_npcap():
         warning("The installed Windump version does not work with Npcap ! Refer to 'Winpcap/Npcap conflicts' in scapy's doc", True)
     del windump_ok
 
+def _test_windows_bluetooth():
+    """Returns wether or not bluetooth is supported on the computer"""
+    query = exec_query(['Get-WmiObject', 'Win32_NetworkProtocol'],
+                           ['Name'])
+    for protocol in query:
```

Comment:
I don't think you need `.strip()` here. FWIW, I would just:
```Python
def _test_windows_bluetooth():
    """Returns wether or not bluetooth is supported on the computer"""
    return any("bluetooth" in protocol.lower() for protocol in
               exec_query(['Get-WmiObject', 'Win32_NetworkProtocol'], ['Name']))
```

Reply:
way more effective, thanks !